tests/events.rs

```diff
@@ -0,0 +1,35 @@
+use std::time::Duration;
+
+use mio::{Events, Poll, Token, Waker};
+
+const WAKE_TOKEN: Token = Token(10);
+
+#[test]
+fn events_all() {
+    let mut events = Events::with_capacity(16);
+    assert_eq!(events.capacity(), 16);
+    assert!(events.is_empty());
+
+    let mut poll = Poll::new().expect("unable to create new Poll instance");
```

Comment:
Small nit; I don't think you need to use `expect`, `unwrap` will do.

Reply:
used init_with_poll.